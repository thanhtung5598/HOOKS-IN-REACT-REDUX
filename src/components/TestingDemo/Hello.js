import React from 'react';
import Link from './Link';

const props = {
    title: 'Default title',
    url: '/link/to/test'
};

export default class Hello extends React.Component {
    render() {
        return(
            <div>
                <h1>
                    Hello
                </h1>
                <Link {...props} />
            </div>
        );
    }
};