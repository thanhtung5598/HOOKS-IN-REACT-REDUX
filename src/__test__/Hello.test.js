import React from 'react';
import Hello from './../components/TestingDemo/Hello';
import { shallowToJson } from 'enzyme-to-json';
import { shallow } from 'enzyme';

it('Render Hello component', () => {
    const wrapper = shallow(<Hello />);
    expect(shallowToJson(wrapper)).toMatchSnapshot();
});