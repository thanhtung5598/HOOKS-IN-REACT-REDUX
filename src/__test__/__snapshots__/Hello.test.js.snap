// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Hello component 1`] = `
<div>
  <h1>
    Hello
  </h1>
  <Link
    title="Default title"
    url="/link/to/test"
  />
</div>
`;
